src/styles.css

```diff
@@ -73,31 +80,63 @@ ul li.active a {
     position: absolute;
     text-align: center;
     top: 8%;
-    padding-left: 45px;
+    padding-left: 3%;
+}
+.title1 h1{
+    font-weight: 440;
+    position: absolute;
+    top: 15%;
+    padding-left: 36%;
+    font-size: 4.3vw;
 }
 
 .title h2{
     color: black;
-    font-size: 70px;
-    line-height: 20mm;
+    font-size: 4.32vw;
+    line-height: 5.5vw;
+}
+
+.text1-container {
+    position: absolute;
+    width: 100%;
+    top: 40%;
+    height: 10%;
+
+    color: black;
+    line-height: 10mm;
+    text-align: center;
+    font-size: 2.5vw;
```

Comment:
Was it a group decision to use viewport elements instead of... say... I dunno... em? Using em - especially for font-size, would make sense, since you want to have several (<5) set font sizes and switch between them - not have a different size for every element.